From 9b1d761206204b82ba51c8fc12a64f3e2493ca2b Mon Sep 17 00:00:00 2001
From: Maitreya29 <maitreyapatni30@gmail.com>
Date: Mon, 21 Feb 2022 10:42:32 +0530
Subject: [PATCH 10/30] Launcher3: Better hotseat searchbar colors

Based on color values from https://github.com/LawnchairLauncher/lawnchair/commit/b69d8d3e941c9e4f0aba43c4683477d206b97d61#diff-25182bc663f7376343f2af1b6646158fcf711a644112d082be074316cc475c5cR75 and thus the additional co author, adapted to our themed only strategy and to use color resources rather than setting them programatically.

Co-authored-by: Suphon Thanakornpakapong <paphonb@gmail.com>
Change-Id: I991e382cc202b4ca46456f3f507f5b28bcdad6b3
---
 res/color-night-v31/qsb_tertiary.xml | 20 ++++++++++++++++++++
 res/color-v31/qsb_primary.xml        | 20 ++++++++++++++++++++
 res/color-v31/qsb_quaternary.xml     | 20 ++++++++++++++++++++
 res/color-v31/qsb_secondary.xml      | 20 ++++++++++++++++++++
 res/color-v31/qsb_tertiary.xml       | 20 ++++++++++++++++++++
 res/values-night-v31/colors.xml      |  1 -
 6 files changed, 100 insertions(+), 1 deletion(-)
 create mode 100644 res/color-night-v31/qsb_tertiary.xml
 create mode 100644 res/color-v31/qsb_primary.xml
 create mode 100644 res/color-v31/qsb_quaternary.xml
 create mode 100644 res/color-v31/qsb_secondary.xml
 create mode 100644 res/color-v31/qsb_tertiary.xml

diff --git a/res/color-night-v31/qsb_tertiary.xml b/res/color-night-v31/qsb_tertiary.xml
new file mode 100644
index 0000000000..ac0a65c79b
--- /dev/null
+++ b/res/color-night-v31/qsb_tertiary.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 The Android Open Source Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@android:color/system_accent2_50" android:lStar="98" />
+</selector>
\ No newline at end of file
diff --git a/res/color-v31/qsb_primary.xml b/res/color-v31/qsb_primary.xml
new file mode 100644
index 0000000000..7a8ad7e5a1
--- /dev/null
+++ b/res/color-v31/qsb_primary.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 The Android Open Source Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@android:color/system_accent3_50" android:lStar="48" />
+</selector>
\ No newline at end of file
diff --git a/res/color-v31/qsb_quaternary.xml b/res/color-v31/qsb_quaternary.xml
new file mode 100644
index 0000000000..638855da0a
--- /dev/null
+++ b/res/color-v31/qsb_quaternary.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 The Android Open Source Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@android:color/system_accent1_50" android:lStar="35" />
+</selector>
\ No newline at end of file
diff --git a/res/color-v31/qsb_secondary.xml b/res/color-v31/qsb_secondary.xml
new file mode 100644
index 0000000000..eec33d14a3
--- /dev/null
+++ b/res/color-v31/qsb_secondary.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 The Android Open Source Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@android:color/system_accent1_50" android:lStar="48" />
+</selector>
\ No newline at end of file
diff --git a/res/color-v31/qsb_tertiary.xml b/res/color-v31/qsb_tertiary.xml
new file mode 100644
index 0000000000..0f51b43fa8
--- /dev/null
+++ b/res/color-v31/qsb_tertiary.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 The Android Open Source Project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@android:color/system_accent2_50" android:lStar="48" />
+</selector>
\ No newline at end of file
diff --git a/res/values-night-v31/colors.xml b/res/values-night-v31/colors.xml
index 541154d9eb..4b4f80b67f 100644
--- a/res/values-night-v31/colors.xml
+++ b/res/values-night-v31/colors.xml
@@ -42,6 +42,5 @@
 
     <color name="qsb_primary">@*android:color/system_accent3_100</color>
     <color name="qsb_secondary">@*android:color/system_accent1_300</color>
-    <color name="qsb_tertiary">@*android:color/system_accent2_500</color>
     <color name="qsb_quaternary">@*android:color/system_accent1_100</color>
 </resources>
\ No newline at end of file
-- 
2.52.0

